tests/test_oneway.py

```diff
@@ -0,0 +1,32 @@
+import multiprocessing
+import thriftpy2
+import time
+from thriftpy2.rpc import make_client, make_server
+
+
+class Dispatcher(object):
+    def Test(self, req):
+        print("Get req msg: %s" % req)
+
+        assert req == "Hello!"
+
+
+oneway_thrift = thriftpy2.load("oneway.thrift", module_name="oneway_thrift")
+multiprocessing.set_start_method("fork")
```

Comment:
This have side effect globally in the lifetime of running all the tests, can we avoid it?

Reply:
You are right, I try to solve this